_pages/openid-connect.md

```diff
@@ -137,6 +137,8 @@ https://idp.int.login.gov/openid_connect/authorize?
    - `address`
    - `email`
    - `phone`
+   - `profile:birthdate`
```

Comment:
IIRC we call it `dob` on the SAML side. It would be nice for consistency in all docs and APIs, and I don't care which one we pick. Fine with me too to merge this as-is and change later, unless you feel like This Is The Moment to clarify.